school/rental.rb

```diff
@@ -0,0 +1,12 @@
+class Rental
+  attr_reader :person, :book
+  attr_accessor :date
```

Comment:
- [ ] Let's add `:person` and `:book` to `attr_accessor` as users can read and write them 👍

```suggestion
class Rental
  attr_accessor :date, :person, :book
```

Reply:
Alright, thanks

Reply:
You're welcome